ansible/group_vars/all.yml

```
@@ -3,7 +3,7 @@
 available_langs: [ ar, de, en, es, sr ]
 
 # root git clone directory (where functionality exists)
-noaa_home: "/home/pi/raspberry-noaa-v2"
+noaa_home: "/home/{{ running_under_user }}/raspberry-noaa-v2"
```

We are still assuming that the playbook is executed by the user we are installing as correct?

In that case I think this would be a better solution:
```suggestion
noaa_home: "{{ lookup('env', 'HOME') }}/raspberry-noaa-v2"
```

---

The alternative if we don't want to make that assumption is to do something fruity like this: https://stackoverflow.com/a/45447488 with getent and set_fact?